sdks/java/core/src/test/java/org/apache/beam/sdk/util/GcsUtilTest.java

```diff
@@ -400,17 +405,30 @@ public void testGetSizeBytesWhenFileNotFoundBatch() throws Exception {
         + "\n"
         + error.toString();
     thrown.expect(FileNotFoundException.class);
-    MockLowLevelHttpResponse notFoundResponse = new MockLowLevelHttpResponse()
-        .setContentType("multipart/mixed; boundary=" + contentBoundary)
-        .setContent(content)
-        .setStatusCode(HttpStatusCodes.STATUS_CODE_OK);
 
+    final LowLevelHttpResponse mockResponse = Mockito.mock(LowLevelHttpResponse.class);
```

Comment:
please do not make an existing test do more things -- each test should only test one thing as much as possible. Instead, create a new test that batch requests are retried.